Jenkinsfile

```diff
@@ -66,6 +66,10 @@ node() {
             sh "cd acceptance-tests && ./run.sh -v=2.107.2 --no-selenium --settings='-s ${env.WORKSPACE}/settings.xml'"
             junit 'acceptance-tests/target/surefire-reports/*.xml'
           }
+          stage('ATH - Jenkins 2.121.1') {
```

Comment:
I assume this is to test all LTS versions still actively supported?
Why is the "latest" being done twice in the case of weekly build?

Reply:
@delitescere I think that alt-job to run the matrix of supported versions isn't used anymore, so this is never used (not really needed, as people tend to stick with LTS, and also the Plugin Compatibility Tester is run by various people regularly anyway, so it is a little redundant). 

To your original point, yes this was to test on all supported versions (not really any hard and fast policy), and it doesn't make sense to run it twice! In the past there was more pressure to run on ye olde cores, but that pressure seems to be less so now (and PCT helps).

Reply:
OK so should I just remove the duplicate for now, or all of it?

It does make sense to test against still-supported LTS versions. What's the oldest LTS version of Jenkins?

Reply:
The stuff inside `if (env.JOB_NAME =~ 'blueocean-weekly-ath') {` is likely not used - I can't see a result in months, so could just remove it I think. 

As for compatibility - maybe @vivek or @olamy knows what is expected? I think it can just be latest LTS, and then trust the PCT will pick up any accidental badness (which is run separately to this pipeline).